lib/rules/no-duplicate-selectors/__tests__/index.js

```diff
@@ -73,6 +73,16 @@ testRule(rule, {
       line: 1,
       column: 1
     },
+    {
+      code: `.a,.b,.a,.c,.d,.a{
```

Comment:
Could you simplify it to the following, please:

```
code: ".a, .b, .a, .c, .d {}"
```

Reply:
done